custom-targets/vertex-ai/advanced_quickstart/ADVANCED_QUICKSTART.md

```diff
@@ -0,0 +1,369 @@
+```
+
+After the post-deploy job has succeeded, you can then inspect the deployed model and view its currently assigned aliases. `dev` and `challenger` should be assigned.
+
+```shell
+gcloud ai models describe test_model@1 --region $REGION --project $PROJECT_ID --format "(versionAliases)"
+```
+
+## 10. Deploy to production
+
+Promote the model in `dev` endpoint to `prod` by using the `gcloud deploy releases promote command`
+
+Since this is the first deployment to that target, we will skipping directly to the `stable` phase, where 100% of the traffic
+goes to the model.
+```shell
+gcloud deploy releases promote --release release-001 --delivery-pipeline=vertex-ai-cloud-deploy-pipeline --project=$PROJECT_ID --region=$REGION --starting-phase-id stable
+```
+
+Monitor the rollout progress using this command:
+
+```
+gcloud deploy rollouts describe release-001-to-prod-endpoint-0001 --release=release-001 --delivery-pipeline=vertex-ai-cloud-deploy-pipeline --project=$PROJECT_ID --region=$REGION --format "(state)"
+```
+
+After the rollout completes, verify that the model has been deployed to the production endpoint:
+
+```shell
+gcloud ai endpoints describe $PROD_ENDPOINT_ID --region $REGION --project $PROJECT_ID
+```
+
+After the post-deploy job has succeeded, inspect the deployed model and view its currently assigned aliases. `prod` and `champion` should be assigned.
+
+```shell
+gcloud ai models describe test_model@1 --region $REGION --project $PROJECT_ID --format "(versionAliases)"
+```
+
+## 11. Upload a new model version and create new release
+
+Create a new version of the model by re-uploading the boston dataset with `gcloud`. 
+This time however, use the `parent-model` flag so that the model is uploaded as
+a new version of the first model rather than a separate model.
+   ```shell
+   gcloud ai models upload \
+       --artifact-uri gs://cloud-samples-data/vertex-ai/model-deployment/models/boston/model \
+       --display-name=test-model \
+       --container-image-uri=us-docker.pkg.dev/vertex-ai-restricted/prediction/tf_opt-cpu.nightly:latest \
+       --project=$PROJECT_ID \
+       --region=$REGION \
+       --parent-model=projects/$PROJECT_ID/locations/$REGION/models/test_model
+   ```
+
+After the upload completes, create a new cloud deploy release using the new model
+version. This time, changing the version ID of the vertexAIModel deploy parameter
+to 2 to refer to the new model.
+
+```shell
+gcloud deploy releases create release-002 \
+    --delivery-pipeline=vertex-ai-cloud-deploy-pipeline \
+    --project=$PROJECT_ID \
+    --region=$REGION \
+    --source=$TMPDIR/configuration \
+    --deploy-parameters="customTarget/vertexAIModel=projects/$PROJECT_ID/locations/$REGION/models/test_model@2"
+```
+
+Inspect the rollout created using the following command:
```

Comment:
The way this is written, it sounds like the rollout was created using the following command.

Either "Inspect the rollout using the following command:"
or "Use the following command to inspect the rollout that was created."